src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -254,6 +263,11 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
     };
   }
 
+  copyEntity(ev) {
+    ev.preventDefault();
+    navigator.clipboard.writeText(ev.model.entity.entity_id);
```

Comment:
There is not full browser support for this API, use <https://github.com/home-assistant/frontend/blob/dev/src/common/util/copy-clipboard.ts> instead. We could add `navigator.clipboard.writeText` in that helper if we check for support first.

Reply:
Fixed and improved.